cxbox-core/src/main/java/org/cxbox/core/dto/rowmeta/RowDependentFieldsMeta.java

```diff
@@ -302,6 +307,12 @@ public final void setDrilldowns(final List<FieldDrillDown> drillDowns) {
 		}
 	}
 
+	/**
+	 * It is not recommended to load images individually for specific values.
+	 * Instead, load all images at once as a complete reference book
+	 * use {@link FieldsMeta#setAllValuesAddIconsLOV}
+	 */
+	@Deprecated(since = "4.0.0-M10")
```

Comment:
@Deprecated method is ready to be deleted

Reply:
done